src/main/kotlin/com/autonomousapps/internal/analyzer/DependencyAnalyzer.kt

```diff
@@ -70,7 +70,8 @@ internal interface DependencyAnalyzer<T : ClassAnalysisTask> {
    * This is a no-op for com.android.application projects, since they have no meaningful ABI.
    */
   fun registerAbiAnalysisTask(
-    dependencyReportTask: TaskProvider<DependencyReportTask>
+    dependencyReportTask: TaskProvider<DependencyReportTask>,
+    abiExclusions: Provider<String>
```

Comment:
I just added support for the java `application` plugin, and so this has changed somewhat on master. FYI.

Reply:
Fixed in my rebase 👍